app/src/main/java/com/simplecity/amp_library/lyrics/QuickLyricUtils.java

```diff
@@ -49,8 +49,7 @@ static Intent getQuickLyricIntent() {
      * @return true if the Play Store is available, and this QuickLyric can be downloaded.
      */
     static boolean canDownloadQuickLyric() {
-        if (ShuttleUtils.isAmazonBuild()) return false;
-        return getQuickLyricIntent().resolveActivity(ShuttleApplication.getInstance().getPackageManager()) != null;
+        return !ShuttleUtils.isAmazonBuild() && getQuickLyricIntent().resolveActivity(ShuttleApplication.getInstance().getPackageManager()) != null;
```

Comment:
I'd prefer not to make this change. 'Simplifying' the conditional statement actually makes it harder to read.